src/EPR.Payment.Facade.UnitTests/Controllers/Payments/PaymentsControllerTests.cs

```diff
@@ -19,15 +18,15 @@ namespace EPR.Payment.Facade.UnitTests.Controllers
     [TestClass]
     public class PaymentsControllerTests
     {
-        private IFixture? fixture;
+        private IFixture? _fixture;
 
         [TestInitialize]
         public void Initialize()
         {
```

Comment:
The initialize method is not needed. There is no point creating a fixture as it is not used.

Reply:
Agreed, I will remove and make sure automoqdata is being used.